app.js

```diff
@@ -216,6 +216,57 @@ if (params) {
   }
 }
 
+
+var isDay = true;
```

Comment:
Please document `isDay`. Also, it would be interesting to move this to `Config.BookViewer`, in file config.js.